.github/workflows/build-resctl-demo.yml

```diff
@@ -6,13 +6,14 @@ on:
   # see https://github.com/iocost-benchmark/iocost-benchmarks-ci/issues/1
   #
   # every Monday at 01:00
+  push:
   schedule:
   - cron: "0 1 * * 1"
   workflow_dispatch:
 
 jobs:
-  run:
-    name: Build resctl-demo
+  build_2_1:
```

Comment:
could this not be done with a matrix instead ?
https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs

Reply:
Great idea!